File: MatMan13_16x64/MatMan_VB/AppCode/ApplicationServer/MatManTypes.vb
Public Enum RefreshType As Integer
	Range = 0
	Sheet = 1
	Workbook = 3
End Enum

Public Enum PlanningFunctionType As Integer
	CostPlan = 1000
	ActivityPlan = 1001
	PurchasingPlan = 1002
End Enum

Public Enum QueryFunctionType As Integer
	GetRequisitionList = 2001
	' associates with WBS item
End Enum

Public Enum FunctionExecutionType As Integer
	RetrievingData = 0
	ValidateData = 1
	ValidateAndPostData = 2
End Enum

File: MatMan13_16x64/MatMan_Lib/AppCode/MatManTypes.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace iiiwave.MatManLib
{
    public enum RefreshType : int
    {
	    Range                         =  0,
	    Sheet                         =  1,
	    Workbook                      =  3
    }

    public enum PlanningFunctionType : int
    {
        CostPlan                      =  1000,
        ActivityPlan                  =  1001,
        PurchasingPlan                =  1002
    }

    public enum QueryFunctionType : int
    {
        GetRequisitionList            =  2001, // associates with WBS item

    }

    public enum FunctionExecutionType : int
    {
        RetrievingData                =  0,
        ValidateData                  =  1,
	    ValidateAndPostData           =  2
    }
}
